src/commands/build.ts

```diff
@@ -31,7 +31,7 @@ export default class Build extends Command {
       );
     }
 
-    const buildCommand = ['build'];
+    const buildCommand = ['build', '--pull'];
     if (flags.parallel) {
```

Comment:
I'd like to see a `pull` flag that defaults to `true` - this way, folks have an escape hatch in case they don't need to hit the network (or can't, if the wi-fi is down).